server/sonar-server/src/main/java/org/sonar/server/branch/ws/ListAction.java

```diff
@@ -97,23 +97,27 @@ public void handle(Request request, Response response) throws Exception {
       checkArgument(project.isEnabled() && PROJECT.equals(project.qualifier()), "Invalid project key");
 
       Collection<BranchDto> branches = dbClient.branchDao().selectByComponent(dbSession, project);
+      Collection<BranchDto> pullRequests = branches.stream().filter(b -> PULL_REQUEST == b.getBranchType()).collect(Collectors.toList());
       Map<String, BranchDto> mergeBranchesByUuid = dbClient.branchDao()
         .selectByUuids(dbSession, branches.stream().map(BranchDto::getMergeBranchUuid).filter(Objects::nonNull).collect(toList()))
         .stream().collect(uniqueIndex(BranchDto::getUuid));
       Map<String, LiveMeasureDto> qualityGateMeasuresByComponentUuids = dbClient.liveMeasureDao()
         .selectByComponentUuidsAndMetricKeys(dbSession, branches.stream().map(BranchDto::getUuid).collect(toList()), singletonList(ALERT_STATUS_KEY))
         .stream().collect(uniqueIndex(LiveMeasureDto::getComponentUuid));
       Map<String, BranchStatistics> branchStatisticsByBranchUuid = issueIndex.searchBranchStatistics(project.uuid(), branches.stream()
-        .filter(b -> b.getBranchType().equals(SHORT))
+        .filter(b -> SHORT.equals(b.getBranchType()) || PULL_REQUEST.equals(b.getBranchType()))
         .map(BranchDto::getUuid).collect(toList()))
         .stream().collect(uniqueIndex(BranchStatistics::getBranchUuid, Function.identity()));
       Map<String, String> analysisDateByBranchUuid = dbClient.snapshotDao()
         .selectLastAnalysesByRootComponentUuids(dbSession, branches.stream().map(BranchDto::getUuid).collect(Collectors.toList()))
         .stream().collect(uniqueIndex(SnapshotDto::getComponentUuid, s -> formatDateTime(s.getCreatedAt())));
 
       ProjectBranches.ListWsResponse.Builder protobufResponse = ProjectBranches.ListWsResponse.newBuilder();
-      branches.forEach(b -> addBranch(protobufResponse, b, mergeBranchesByUuid, qualityGateMeasuresByComponentUuids.get(b.getUuid()), branchStatisticsByBranchUuid.get(b.getUuid()),
+      branches.stream()
+        .filter(b -> LONG == b.getBranchType() || SHORT == b.getBranchType())
+        .forEach(b -> addBranch(protobufResponse, b, mergeBranchesByUuid, qualityGateMeasuresByComponentUuids.get(b.getUuid()), branchStatisticsByBranchUuid.get(b.getUuid()),
           analysisDateByBranchUuid.get(b.getUuid())));
+      pullRequests.forEach(pr -> addPullRequest(protobufResponse, pr, mergeBranchesByUuid, branchStatisticsByBranchUuid.get(pr.getUuid()), analysisDateByBranchUuid.get(pr.getUuid())));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Split this 184 characters long line (which is greater than 180 authorized). ([squid:S00103](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fteryk%2Flist-pr&issues=AWF8sv0SHTiOwYNMzcIt&open=AWF8sv0SHTiOwYNMzcIt)

[ISSUE_KEY]: AWF8sv0SHTiOwYNMzcIt